install_config/install/docker_registry.adoc

```diff
@@ -169,9 +170,68 @@ $ oadm registry --service-account=registry \
 ----
 endif::[]
 
-[[viewing-logs]]
 
+[[registry-ip-address]]
+=== Registry IP Address
```

Comment:
Can this have a more descriptive title? Something like "Defining the Registry IP Address" maybe? Something that describes the process the reader is about to undertake.